app/views/home/_contact_text.slim

```diff
@@ -0,0 +1,4 @@
+== t('home.send_us_an_email', mail_to: mail_to(Whitelabel[:email], 'E-Mail', title: 'E-Mail'))
+- if Whitelabel[:twitter]
+  ==< t('home.or_message_on_twitter', twitter_link: link_to_twitter(Whitelabel[:twitter]))
+| .
```

Comment:
IMHO: this should be a helper method, instead of a partial. Performance of many small partials isn't great.

Reply:
template is still included.